.changeset/clean-glasses-walk.md

```diff
@@ -0,0 +1,6 @@
+---
+"vue-demo-store": minor
+"@shopware-pwa/cms-base": minor
```

Comment:
```suggestion
```
this is not the change in cms package, we wouldn't want that kind of logic there :)